go_sequencer/common/tx.go

```diff
@@ -75,25 +75,13 @@ const (
 	// TxTypeExit represents L2->L1 token transfer.  A leaf for this account appears in the exit
 	// tree of the block
 	TxTypeExit TxType = "Exit"
-	// TxTypeTransfer represents L2->L2 token transfer
-	TxTypeTransfer TxType = "Transfer"
```

Comment:
Just out of curiosity, does this mean we don't support `Transfer` in the initial version? 🤔

Reply:
Yeah, @Intizar-T We are not supporting transfer in the initial version